src/chrome/content/rules/uni-mannheim.de.xml

```diff
@@ -0,0 +1,31 @@
+<ruleset name="Universität Mannheim">
```

Comment:
Please add ` (partial)` to the ruleset name, since this ruleset covers only a small fraction of `*.uni-mannheim.de`. Please read our docs, especially [this](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains) section. You can easily find more hosts. At least the most important/popular/sensitive ones should be added.